api/candlepin-api-spec.yaml

```diff
@@ -555,6 +555,57 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /cloud:
+    post:
+      description: Verifies provided cloud registration data and returns an authentication token. By default the token is valid for 10 minutes
+      tags:
+        - Cloud registration
+      operationId: cloudAuthorize
+      security: [ ]
+      requestBody:
+        description: Cloud registration data
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CloudRegistrationDTO'
+      responses:
+        200:
+          description: Cloud authorization successful.
+          content:
+            text/plain:
+              schema:
+                type: string
+        400:
+          description: CloudRegistrationInfo is null
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: CloudRegistrationInfo is null
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        401:
+          description: Cloud provider or account details could not be resolved to an organization
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Cloud provider or account details could not be resolved to an organization
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        501:
+          description: Cloud provider or account details could not be resolved to an organization
```

Comment:
An error message is not updated here after pasting the error block. The example shows the correct error message.

Reply:
Done.